### PR TITLE
Seeds of Inception - Part 7

### PR DESCRIPTION
**Pull request by mechanicalpete**
How to enable Global CloudTrail in your AWS Organization.

### CODE REVIEW

File: _posts/2019-07-07-inception-pipelines-pt7.markdown [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+Typically you'll see [AWS CloudTrail](https://aws.amazon.com/cloudtrail/) enabled within an account and recording events into an [S3 Bucket](https://aws.amazon.com/s3/). When it is done this way though, there is always a possibility that a rogue actor, with the right permissions, can tampered with it. However [AWS CloudTrail](https://aws.amazon.com/cloudtrail/) can also be enabled through [AWS Organizations](https://aws.amazon.com/organizations/). This means you can pick your Security/Audit account to receive **ALL** the trail events and the individual accounts don't get a chance to tamper with them as they never see the trail. You even get the additional benefit that any new accounts that get added to your [AWS Organizations](https://aws.amazon.com/organizations/) automatically start recording events into the bucket.
```
Comment (Hamo69) [inline _posts/2019-07-07-inception-pipelines-pt7.markdown]: `can tampered with it` should be `can tamper with it`

File: _posts/2019-07-07-inception-pipelines-pt7.markdown [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+When I asked around about this feature no one else knew about it! Big shout out to James for pointing me in the right direction!
```
Comment (Hamo69) [inline _posts/2019-07-07-inception-pipelines-pt7.markdown]: surname? Add Bromberger or a link to his profile?

File: _posts/2019-07-07-inception-pipelines-pt7.markdown (hunk 1 above)
Comment (mechanicalpete) [reply _posts/2019-07-07-inception-pipelines-pt7.markdown]: Fixed

File: _posts/2019-07-07-inception-pipelines-pt7.markdown (hunk 2 above)
Comment (mechanicalpete) [reply _posts/2019-07-07-inception-pipelines-pt7.markdown]: Add a link to James' LinkedIn profile.

File: _posts/2019-07-09-inception-pipelines-pt7.markdown [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+4. Deploy the pipeline and wait for it to complete. Once it has you will need to extract the following values from the infrastructure stack. These will be used when configuring the CloudTrail trail in the next section:
+
+    | Name       | Description                                                       |
+    | ---------- | ----------------------------------------------------------------- |
+    | BucketName | The complete name of the S3 bucket where we are storing the logs. |
+    | KmsKeyArn  | The KMS Key ARN to use to encrypt the logs.                       |
+
+## Enable AWS Organizations Global CloudTrail
+
+Now that the underlying infrastructure is deployed, we can start on ~~world domination~~ configuring Global [CloudTrail](https://aws.amazon.com/cloudtrail/)! Since this is a once-off activity it is easier and quicker to deploy via the console; one of the ultra-rare times that I will use the console for deployments.
```
Comment (temyers) [inline _posts/2019-07-09-inception-pipelines-pt7.markdown]: Is there a CLI alternative?

**Comment by temyers** [comment]
Does this deserve a ControlTower mention - if only to say you're targetting the post at people that already have AWS accounts set up.

File: _posts/2019-07-09-inception-pipelines-pt7.markdown (hunk 3 above)
Comment (mechanicalpete) [reply _posts/2019-07-09-inception-pipelines-pt7.markdown]: There is a CLI alternative; however for this post I chose to focus on the UI approach.

**Pull request merged**